umbra-js/src/classes/Umbra.ts

```diff
@@ -391,7 +397,25 @@ export class Umbra {
   }
 
   /**
-   * @notice Fetches all Umbra event logs using The Graph
+   * @notice Fetches Umbra event logs starting from the block user registered their stealth keys in using Goldsky, if available, falling back to RPC if not
```

Comment:
Line length should be capped at 120 chars

Reply:
Broke it up here: https://github.com/ScopeLift/umbra-protocol/pull/549/commits/222d7af6be812c9cf7f8d63de8b8413b75bdd7c3